README.md

```diff
@@ -8,18 +8,18 @@ For more detailed information on the server and API design refer to the [wiki](h
 
 ## Supported NI drivers
```

Comment:
My only remaining question about the readme is whether driver support is meant to apply to the source code or the latest release, and whether it's necessary to clarify that.

Reply:
@astarche, @DavidCurtiss, @reckenro - any thoughts on this? Maybe this documentation shouldn't be updated until next release..?

Reply:
@Wedgetable that's a good question. We have always made the update to this table immediately before release to avoid that confusion. Also, previously, our releases have been totally organized around adding driver support. That's not as much the case now; and we have more internal clients working off grpc-device artifacts that are not from the GH release.

Technically it is fine/typical for the github readme refer to the source for whatever branch it's in. Most open source projects do this, and many have a notice saying so. But, certainly it can also cause confusion if that's different than the latest download/package/wahtever, especially since we don't have a "released" version of this table that's easy to refer to.

Could someone track completing this PR as a release task for the next grpc-device release? If so, I think that's a good option.

Otherwise, I think you should go ahead and complete the PR. And we should start including the version table in the release notes.

But I'm not involved in the grpc-device release process anymore, so I'm fine being overruled.

Reply:
Sorry for the late reply to this.

Personally, I like the model of the readme being up to date with source and including the version table (or a version of it) in the Release notes to be accurate for that release.

I'll pull in our PO, @harsha-bhushan, to see his thoughts on it.

Reply:
I agree with the proposal. Let's do these right away-
1.Update the table on the github page
2.Include readme in the client & server artefacts going ahead (We can submit the changes to the release process but shouldn't retrospectively change already published releases & the latter should not be a big problem)
3.Add a note above the table that the online page reflects the driver support versions of the source & the one downloaded along with the release reflects that of the release

Regarding the terminology 'Minimum' I'd like to discuss it more in our internal channel & then update it accordingly.

Reply:
As far as RFmx personalities, update the versions for only those personalities that we have updated & tested.

Reply:
I added a note, let me know if it sounds accurate. The versions I changed in this PR have all already been explicitly updated to 22.5 and tested.